blark/iec.lark

```diff
@@ -722,6 +722,7 @@ _statement: ";"
           | reset_statement
           | reference_assignment_statement
           | return_statement
+          | function_call_statement_list
```

Comment:
I'll be a bit nitpicky here with naming - how about `chained_function_call_statement`?

Reply:
I had *very strongly* considered that! :laughing: I thought that maybe I'd stick with "list" just because that felt more in-line with other things, but I'm for it! I'll make that change...